templates/versafix-1/template-versafix-1.html

```diff
@@ -1621,16 +1621,17 @@
                            width="100%" style="display:none; width: 100%; height:100%">
                         <tr>
                           <td valign="middle" align="center"
+                              class="long-text links-color mobile-textcenter"
+                              data-ko-editable="longText"
                               style="font-size: 13px; font-family: Arial, Helvetica, sans-serif; color: #3f3f3f;text-align:center;
-                              -ko-font-size: @[longTextStyle.size]px; -ko-font-family: @longTextStyle.face; -ko-color: @longTextStyle.color;height:100%;"
```

Comment:
How that is related? You just disabled to control fonts of the block from the editor